agent/pkg/dagent/utils/docker.go

```diff
@@ -643,7 +657,7 @@ func GetImageLabels(fullyQualifiedImageName string) (map[string]string, error) {
 	if res.Config != nil && res.Config.Labels != nil {
 		return res.Config.Labels, err
 	} else {
-		return nil, errors.New("no labels")
+		return map[string]string{}, errors.New("no labels")
```

Comment:
Is it an error, though? Having no labels on an arbitrary image is a state being completely normal.